tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters.bal

```diff
@@ -195,6 +199,10 @@ function functionOfFunctionTypedParamWithIncludedRecordParam22(*Pairs values) re
     return <float[]>values["a"];
 }
 
+function functionOfFunctionTypedParamWithIncludedRecordParam23(*Baz baz, *Baz2 baz2) returns [anydata, anydata] {
```

Comment:
What does `functionOfFunctionTypedParam` here mean?

Reply:
I have renamed this. I used this because I used a similar name for previous functions. I think we need to rename all of those function names.